great-idea-website/index.html

```diff
@@ -14,50 +14,71 @@
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-  
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
-  
-  </body>
-</html>
\ No newline at end of file
+<body>
+  <nav>
+    <a href="#">Services</a>
+    <a href="#">Product</a>
+    <a href="#">Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+
+  <header>
+    <h1>Innovation<br>On<br>Demand</h1>
+    <img src="img/header-img.png" alt="Image of a code snippet.">
+  </header>
+  <button>Get Started</button><br><br>
+
+  <hr>
+
+  <div class="div1">
+    <h3>Features</h3>
+    <p>
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm
+      nisi.Suspendisse
+      eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+    <h3>About</h3>
+    <p>
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse
+      eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+  </div>
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen" width="100%;">
+
+  <div class="div2">
+    <h3>Services</h3>
+    <p id="p1"> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm
+      nisi.
+      Suspendisse
+      eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
```

Comment:
Id's are not necessary here, ids are generally used to target a specific element in javascript. Try to use classes for styling. 

In this case as you are applying the same styling to a all three paragraphs you can simply select them like so:

.div2 p {
 /* your styles */
}

With no need for additional classes or ids.